toolchain/DetectLinux.cmake

```diff
@@ -0,0 +1,22 @@
+# Parse /etc/os-release for Linux version information
```

Comment:
@GMKennedy this file was supposed to have been removed in our last refactor as it's no longer used.

Reply:
Cmake Toolchain has been updated, DetectLinux.cmake has been removed change verified in test environments.